### ISSUE TITLE
Enable practicing of shortcuts that include mouse clicks

### ISSUE THREAD

**Issue by tkainrad** [tkainrad/keycombiner#64]
Requested via mail

It is possible to define shortcuts like <kbd>Ctrl</kbd>+<kbd>Click</kbd> in personal collections, but these are excluded from practice as <kbd>Click</kbd> is not a [valid key](https://github.com/tkainrad/keycombiner/wiki/Valid-Key-Combinations).

It would be cool if there was a way to practice these shortcuts. For example, KeyCombiner could wait for the user to click on some specific part of the screen.

**Comment by tkainrad** [comment]
A related request via mail would be to show the mouse buttons also in the collection visualizer.

**Comment by HarryLang100** [comment]
I agree with this request. Just the other day, I studied the [Slack public collection](https://keycombiner.com/collections/slack/). One of the shortcuts was `alt` + `click`. I had to remove this. So now my collection is incomplete.

**Comment by tkainrad** [comment]
I am currently evaluating the roadmap for the next months, so it's good to know that you would also be interested in this, thx :+1: 

(I am not making any promises though. There is so much interesting stuff to do :see_no_evil: )

**Comment by cooperd1958** [comment]
I agree.  Photoshop uses a lot of clicks.  As a workaround I replace ckick with space and double-click with space > space.

**Comment by TheHipshaker** [comment]
I would even go further and would like to add some more mouse buttons and the wheel. Are there any updates on this issue?
